src/pages/collection-details/collection-details.css

```diff
@@ -0,0 +1,98 @@
+.collection {
+  margin: auto;
+  width: 70%;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.collection__top-details {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
+}
+
+.collection__images {
+  height: 150px;
+  width: 150px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  border: 2px solid var(--purple);
+  flex-shrink: 0;
+}
+
+.collection__images > img {
+  width: 30%;
+  object-fit: fill;
+  margin: 2px;
+}
+
+.collection__progress {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+  margin-left: 30px;
+  width: 100%;
+  height: 150px;
+}
+
+.collection__progress_name {
+  align-self: start;
+  flex-grow: 1;
+  font-weight: bold;
+}
+
+.collection__progress_text {
+  align-self: start;
+  margin-bottom: 10px;
+  margin-left: 5px;
+}
+
+#collection__progress_bar {
+  background-color: black;
+  border-radius: 15px;
+  /* (height of inner div) / 2 + padding */
```

Comment:
o co chodzi w tym komentarzu?

Reply:
To jest komentarz do ustawienia border-radius na tym czarnym elemencie. Ale nieistotny. Wyrzuciłem.